js/Source/FlatUI.Select/Behavior.FlatUI.Select.js

```diff
@@ -14,32 +14,49 @@ provides: [Behavior.FlatUI.Select]
 ...
 */
 
-Behavior.addGlobalFilter('FlatUI.Select', {
+(function(){
+
+// the filter for both FlatUI.Select and FlatUI.Select.Filter
+// is identical; the only difference is which class is used
+var init = function(el, api, filterable){
+  var cls = filterable ? FlatUI.Select.Filter : FlatUI.Select;
```

Comment:
i don't know if `cls` is standard in js, but in ruby, we use the word `klass`. it's clearer.